### DISCUSSION TITLE
MediaCreationToolW11.exe no longer a valid link

### DISCUSSION THREAD

**Discussion by UltimateBoi** [AveYo/MediaCreationTool.bat#10] [Q&A] [answered]
Title ^

**Comment by AveYo** [comment]
How come?
It does not download it anymore for you?
Because it works fine here, the url has not changed

**Reply by UltimateBoi** [reply]
It dosent work for me

**Reply by UltimateBoi** [reply]
![image](https://user-images.githubusercontent.com/63293118/138412504-dd8f48df-6207-4cd8-b283-49d383fa8900.png)

**Reply by UltimateBoi** [reply]
nvm it just dosent work and comes out with that error

**Comment by UltimateBoi** [comment]
I had to manually download it from the batch code to download W11 media creation tool exe but it still  dosent bypass the tpm check

**Comment by AveYo** [comment, marked as answer]
I have finally improved the download function, it will try different methods as a fallback
TPM bypassing is still not ideal, but should work if you use `auto.cmd` to run setup

**Comment by NOTAndry** [comment]
where is the download?

**Reply by AveYo** [reply]
Sometimes it can be unavailable, it's a microsoft servers issue.
Try again later and it comes back up. Right now the link works.

**Reply by NOTAndry** [reply]
ok

**Comment by RoyalGame97** [comment]
Hello, thank you very much, this is a huge work of innumerable labor, but I really need your help to use your work, I want to run your script in the domain, but since I am from Russia, Microsoft blocked these links and I have the same problem, are there any ways to get around this problem without using VPN? Please help :(
